### PR TITLE
Notebook template

### PR DESCRIPTION
**Pull request by sudarshan-SpringML**
This PR contains a small fix(adding Vertex AI link at top) for notebook_template.ipynb

### CODE REVIEW

File: notebooks/notebook_template.ipynb [hunk 1]
```diff
@@ -1,701 +1,725 @@
...
+    "    </a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a href=\"https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/master/notebooks/notebook_template.ipynb\">\n",
+    "      <img src=\"https://cloud.google.com/ml-engine/images/github-logo-32px.png\" alt=\"GitHub logo\">\n",
+    "      View on GitHub\n",
+    "    </a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a href=\"https://console.cloud.google.com/vertex-ai/workbench/deploy-notebook?download_url=https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/master/notebooks/notebook_template.ipynb\">\n",
```
Comment (kweinmeister) [inline notebooks/notebook_template.ipynb:46]: Can you please add the Vertex AI image for consistency with the other links:

\n"

File: notebooks/notebook_template.ipynb [hunk 2]
```diff
@@ -1,701 +1,725 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "JAPoU8Sm5E6e"
+   },
+   "source": [
+    "<table align=\"left\">\n",
+    "\n",
+    "  <td>\n",
+    "    <a href=\"https://colab.research.google.com/github/GoogleCloudPlatform/vertex-ai-samples/blob/master/notebooks/notebook_template.ipynb\">\n",
```
Comment (kweinmeister) [inline notebooks/notebook_template.ipynb:35]: Let's change master to main here while we're at it.

File: notebooks/notebook_template.ipynb [hunk 3]
```diff
@@ -1,701 +1,725 @@
...
+    "      <img src=\"https://cloud.google.com/ml-engine/images/colab-logo-32px.png\" alt=\"Colab logo\"> Run in Colab\n",
+    "    </a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a href=\"https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/master/notebooks/notebook_template.ipynb\">\n",
```
Comment (kweinmeister) [inline notebooks/notebook_template.ipynb:40]: Another place to update the URL to main.

File: notebooks/notebook_template.ipynb (hunk 1 above)
Comment (kweinmeister) [reply notebooks/notebook_template.ipynb:46]: Also, let's update the URL to use main

**Review by kweinmeister — CHANGES_REQUESTED** [review]

File: notebooks/notebook_template.ipynb (hunk 3 above)
Comment (sudarshan-SpringML) [reply notebooks/notebook_template.ipynb]: all comments resolved

**Comment by kweinmeister** [comment]
Thanks @sudarshan-SpringML !

**Review by kweinmeister — APPROVED** [review]

**Pull request merged**